enterprise/server/backends/migration_cache/migration_cache.go

```diff
@@ -1025,6 +1025,17 @@ func (mc *MigrationCache) copy(c *copyData) {
 		log.CtxDebugf(ctx, "Migration already copied on dequeue, returning early: digest %v", c.d)
 		return
 	}
+	if c.d.GetDigest().GetSizeBytes() > 100 && c.conf.src.SupportsCompressor(repb.Compressor_ZSTD) && c.conf.dest.SupportsCompressor(repb.Compressor_ZSTD) {
```

Comment:
I like the idea behind this change. Make `c.d.GetDigest().GetSizeBytes() > 100` a util function in case we change the flag's value?